src/utils/events/event-manager.js

```diff
@@ -1,13 +1,15 @@
-import {Manager} from 'hammerjs';
+import WheelInput from './wheel-input';
+import MoveInput from './move-input';
 
-import {
+const Manager = (typeof window === 'undefined' || typeof document === 'undefined') ?
```

Comment:
We have a function in utils that is intended to do a safe check for whether we are running under node.js (a check that is not fooled e.g. by jsdom inclusion). Consider using that?

Also, add a short comment?